src/test/scala/org/broadinstitute/dsde/vault/services/uBAM/UBamDescribeServiceSpec.scala

```diff
@@ -8,9 +8,9 @@ import spray.http.HttpHeaders.Cookie
 import spray.http.StatusCodes._
 import spray.httpx.SprayJsonSupport._
 
-class UBamDescribeServiceSpec extends VaultFreeSpec with UBamDescribeService with UBamIngestService {
+class UBamDescribeServiceSpec extends VaultFreeSpec with UBamDescribeService with UBamIngestService with UBamDescribeListService{
 
-  override val routes = uBamDescribeRoute
+  override val routes = uBamDescribeRoute;  uBamDescribeListRoute
```

Comment:
A good point!  Yes, I like this idea.  Let's refactor all these `routes` `val`s to unique values, and this should not be necessary.

Added https://broadinstitute.atlassian.net/browse/DSDEV-1945